src/optimizer/asp_optimizer.cpp

```diff
@@ -25,54 +25,67 @@ void ASPOptimizer::visitOperator(planner::LogicalOperator* op) {
 
 void ASPOptimizer::visitHashJoin(planner::LogicalOperator* op) {
     auto hashJoin = (LogicalHashJoin*)op;
-    if (!canApplyASP(op)) {
+    if (hashJoin->getChild(0)->getOperatorType() == LogicalOperatorType::ACCUMULATE) {
+        // No ASP if probe side has already been accumulated. This can be solved.
```

Comment:
What does "This can be solved." here mean? You mean this can be ignored?

Reply:
Meaning I'll gradually remove these constraints.